support/docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+networks:
+  netpanzer-net:
+    external: false
+
+services:
+  netpanzer-server:
+    container_name: netpanzer-server
+    ports:
+      - 3030:3030/tcp
```

Comment:
I think it should be possible to use environmental variables so some options in this file will be more easily configurable.